app/build.gradle.kts

```diff
@@ -65,15 +65,18 @@ dependencies {
     androidTestImplementation(libs.androidx.ui.test.junit4)
     debugImplementation(libs.androidx.ui.tooling)
     debugImplementation(libs.androidx.ui.test.manifest)
-    implementation("com.squareup.okhttp3:okhttp:4.12.0")
-    implementation ("androidx.appcompat:appcompat:1.2.0")
-    implementation ("com.squareup.retrofit2:retrofit:2.11.0")
-    implementation ("com.squareup.retrofit2:converter-gson:2.11.0")
-    implementation ("com.squareup.okhttp3:okhttp:4.12.0")
-    implementation( "com.github.bumptech.glide:glide:4.15.0")
-    annotationProcessor ("com.github.bumptech.glide:compiler:4.15.0")
-    implementation ("androidx.recyclerview:recyclerview:1.2.1")
-    implementation ("com.google.android.material:material:1.2.1")
+    implementation(libs.okhttp3.okhttp)
+    implementation (libs.androidx.appcompat.v120)
+    implementation (libs.retrofit)
+    implementation (libs.converter.gson)
+    implementation (libs.okhttp3.okhttp)
+    implementation( libs.glide)
+    annotationProcessor (libs.compiler)
+    implementation (libs.androidx.recyclerview.v121)
+    implementation (libs.material.v121)
+    implementation(libs.kotlinx.coroutines.core)
+    implementation (libs.androidx.navigation.fragment.ktx)
+    implementation (libs.androidx.navigation.ui.ktx)
```

Comment:
Add comments to clarify the categories here please. 
//Network
implementation(libs.okhttp3.okhttp)
implementation (libs.retrofit)
implementation (libs.converter.gson)

//Coroutines